apache/superset
# feat: Move cross filters to Dashboard

### SUMMARY
This PR does the following:

- Removes the ability to emit cross filters from Explore
- Moves the emission of cross filters at the Dashboard level
- Removes the cross filter target name introduced in https://github.com/apache/superset/issues/13966
- Moves the filter bar location settings in a submenu

### AFTER

https://user-images.githubusercontent.com/60598000/213498463-4b990134-247b-4033-9061-cbd617e7be13.mp4

### TESTING INSTRUCTIONS
1. Go to a Dashboard
2. Open the filter bar on the left
3. Click on the settings icon
4. Make sure the checkbox to enable/disable cross filters is appearing
5. Verify that cross filters are enabled by default (requires Feature Flag DASHBOARD_CROSS_FILTERS)
7. Disable cross filtering and make sure the change takes effect
8. Verify that the filter bar orientation settings are appearing in a submenu and working as intended

### ADDITIONAL INFORMATION


- [x] Required feature flags: DASHBOARD_CROSS_FILTERS
